app/jsx/components/TabContent2Comp.jsx

```diff
@@ -1,15 +1,81 @@
-// ##### Tab Content 2 Component ##### //
+// ##### Tab Content 2 Component - Supplemental Materials ##### //
```

Comment:
On your todo list for the future, would you consider renaming TabContent2Comp to TabSuppComp, and likewise with Tab1Content, Tab3Content, etc.? It'd help when trying to find things.